DBTEMPLATE/post-snapshot/scripts/dxh_wrapper_script.sh

```diff
@@ -39,14 +39,7 @@ HOOK_OP=${4}
 #
 
 # sample code to initiate impdp
-#/softwares/scripts/hooks/dxh_impdp.sh -o ${HOOK_OP} -d '/softwares/scripts/hooks/dumpdir' -p DBTEMPLATE.impdp.delphix.par 
-
-
-
-
-
-
-
-
-
+#/home/delphix/dxhooks/dxh_impdp.sh -o ${HOOK_OP} -d '/softwares/scripts/hooks/dumpdir' -p DBTEMPLATE.impdp.delphix.par 
+###script for Archivelog deletion older than 7 days. Enter VDB name in DBNAME in RMAN_AL_deletion.sh script, you can change number of days in the script
+/home/delphix/dxhooks/RMAN_AL_deletion.sh -o ${HOOK_OP}
```

Comment:
Please add a comment to line 44, so that this requires the user to uncomment it in order for it to run.  That's the basic idea behind using dxhooks.